templates/archives/stories/women_in_symbols.jinja2

```diff
@@ -0,0 +1,39 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+Women in Symbols
+{% endblock %}
+
+{% block story_body %}
+<p>Until recently, women have gone unnamed in many of their historical contributions, which is visible in this archive.
+    Most of the historical materials we can reference here to learn about early computing have been signed by men, with little evidence of the women behind them.
+    For example, a majority of the letters in this archive have been written by a secretary, whose initials only appear at the end of the letter, such as in the picture to the right, where FMV has signed the letter and NB has written it up.</p>
+
+<figure class="figure float-right">
+    <img src="{{ static('img/stories/women_in_symbols.png') }}"
+         class="class="figure-img img-fluid rounded"
+         alt="A letter subscript above the cc line detailing those involved in its writing" />
+</figure>
+
+<p>However, internal documents sometimes display a portion of the work women performed in computing.
+    In the case of the choices made for which symbols computers would use, this archive demonstrates that women were heavily involved.
+    In a memo submitted to Laura by Edna Verzuh, there is a page of symbols which appear to await her commentary.
+    This is precedented by the file following this memo, in which there are comments on the symbols requested by Edna herself, which are marked by both her name on the sheet and her initials, ETV, in the bottom right-hand corner, as well as comments by Nanny, marked also by her name and her initials, NB, on the bottom left-hand corner. This Nanny is also likely Frank Verzuh’s secretary, seeing as the secretary subscript on most of Verzuh’s outgoing correspondence is nb.
+    This is one example of the many tasks secretaries completed which diverged from the tasks we typically associate with them, and portrays how secretaries were and are far more than typists and organizers.</p>
+
+<p>After the memo was received by Laura, final comments seem to have been sent by her to Frank Verzuh in a note requesting the agreed upon new symbols, although the initials on the bottom of this note are too ambiguous to say with certainty.</p>
+
+    <p>Through these documents, we are able to view a piece of the role women played in the administrative details regarding computing, as well as some of the decision making and female labor behind the MIT Computation Center.
+        Secretaries, which have often been credited only as typists, were often responsible for work relying on their judgement and decision making as well.
+        This trend is also present in scholarly manuscripts, as shown in <i>#ThanksForTyping Spotlights Unnamed Women In Literary Acknowledgments</i>, an NPR article discussing the female labor behind many scholarly publications.
```

Comment:
I would say that "This trend is present across scholarly disciplines" and then include the link `[#ThanksForTyping](https://www.npr.org/2017/03/30/521931310/-thanksfortyping-spotlights-unnamed-women-in-literary-acknowledgements)`. The Computation Center is one part of a larger academic ecosystem that is only crediting some of its laborers and hiding the contributions of others.